docker/monitoring/docker-compose.monitoring.yml

```diff
@@ -1,35 +1,6 @@
 ---
 version: '3.8'
 services:
-  datahub-gms:
-    environment:
-      - ENABLE_PROMETHEUS=true
-      - ENABLE_OTEL=true
-      - OTEL_TRACES_EXPORTER=jaeger
-      - OTEL_EXPORTER_JAEGER_ENDPOINT=http://jaeger-all-in-one:14250
-      - OTEL_METRICS_EXPORTER=none
-      - OTEL_SERVICE_NAME=datahub-gms
-    ports:
-      - "4318"
-
-  datahub-frontend-react:
-    environment:
-      - ENABLE_PROMETHEUS=true
-      - ENABLE_OTEL=true
-      - OTEL_TRACES_EXPORTER=jaeger
-      - OTEL_EXPORTER_JAEGER_ENDPOINT=http://jaeger-all-in-one:14250
```

Comment:
Open telemetry is a separate agent we add during runtime to support jaeger/tracing. The metrics change shouldn't impact this.